apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,27 @@
 # API Changelog
 
+## BFD-1424 Fix mtus code
+
+Update Carrier claim mapping for mtus code instead of mtus count:
+The new coding:
+```
+{
+      "url" : "https://bluebutton.cms.gov/resources/variables/carr_line_mtus_cnt",
+      "valueCoding" : {
+        "system" : "https://bluebutton.cms.gov/resources/variables/carr_line_mtus_cnt",
+        "code" : "3"
+      }
+},
+{
+      "url" : "https://bluebutton.cms.gov/resources/variables/carr_line_mtus_cd",
+      "valueCoding" : {
+        "system" : "https://bluebutton.cms.gov/resources/variables/carr_line_mtus_cd",
+        "code" : "3",
+        "display":"Services"
+      }
+}
+```
+
```

Comment:
I feel like we should probably mention and show the details of the element that we are leaving behind for backwards compatibility. thoughts @bfausett77 ?

Reply:
That makes sense.  This is more clear and 'feels' more appropriate than adding an extension to 'deprecate' the element that we are leaving behind.  Great suggestion Trotter!